subsys/can/isotp/Kconfig

```diff
@@ -0,0 +1,129 @@
+# Kconfig - ISO TP configuration options
+
+#
+# Copyright (c) 2019 Alexander Wachter
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig ISOTP
+	bool "ISO-TP"
+	select NET_BUF
+	select POLL
+	help
+	  Enable ISO TP support for CAN
+
+if ISOTP
+
+module = CAN_ISOTP
+module-str = CAN_ISOTP
+source "subsys/logging/Kconfig.template.log_config"
+
+config CAN_ISO_TP_WFTMAX
+	int "WFTmax"
```

Comment:
I wonder if we should open these acronyms here and the other options below a bit. So here could we say something like "Max wait frames before abort (WFTmax)" instead, I do not think that would be too long string to show here?
And similar treatment to the other option texts.

Reply:
These namings are actually from the standard. IMO when someone uses ISO-TP, he should know what they do. But maybe some more text would be good still.